### ISSUE TITLE
 tf.train.match_filenames_once() under Windows

### ISSUE THREAD

**Issue by warmestwind** [tensorflow/tensorflow#15329]
I use  tf.train.match_filenames_once() as follows:
 tf.train.match_filenames_once("./*.py")
I mean , I want to get all the py file in current directory.
But, the result is I also get subdirectory py file under Windows.
How can I get the specified files only in current directory?
Thank you！
-----------------------
Have I written custom code : No
OS Platform and Distribution: Windows
TensorFlow installed from :pip command
TensorFlow version: 1.4
Bazel version: N/A
CUDA/cuDNN version: CUDA8.0
GPU model and memory:  GTX1080 8G
Exact command to reproduce： tf.train.match_filenames_once("./*.py")

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by Carmezim** [comment]
Thank you for your interest in TF. This question is better suited for StackOverflow, which the TensorFlow team also monitors under the TensorFlow tag. Github is strictly for bugs and feature requests.

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue? Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Awaiting Response: It has been 14 days with no activityand the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
Nagging Awaiting Response: It has been 14 days with no activityand the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
Nagging Awaiting Response: It has been 14 days with no activityand the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
Nagging Awaiting Response: It has been 14 days with no activityand the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
It has been 29 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
It has been 152 days with no activity and the `awaiting response` label was assigned. Is this still an issue?
